### PR TITLE
Create Developer - Data Services.MD

### PR DESCRIPTION
**Pull request by rachellegupta**
Position description for the Join 18F site for DAP  - Developer Data Services

### CODE REVIEW

**Comment by fureigh** [comment]
Hi @rachellegupta,

Same initial remarks as on #319. Thanks!

**Comment by JenTress** [comment]
@fureigh and @rachellegupta same as #319. Think we're good to go!

**Comment by rachellegupta** [comment]
made the recommended changes. Thanks @fureigh

File: _positions/developer-data-services.MD [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+title: Developer 
+permalink: _positions/developer-data-services.MD
+team: gsa feedback analytics
+active: true
+description: Job posting for the GSA Feedback Analytics Program Manager.
+---
+
+*Please note: This role is for the General Services Administration's Feedback Analytics Program, not 18F.*
+
+#Program Manager
```
Comment (fureigh) [inline _positions/developer-data-services.MD:11]: @JenTress Can you confirm that this is the intended job title? From the filename I'd assume it was `Data Services Developer`.

Comment (fureigh) [inline _positions/developer-data-services.MD:11]: cc @rachellegupta

Comment (rachellegupta) [reply _positions/developer-data-services.MD:11]: Developer - Data Services @fureigh

**Pull request merged**